developers/madara-integration.md

```diff
@@ -0,0 +1,85 @@
+# Integrating Celestia with Madara
+
+[Celestia](https://celestia.org/) is a modular data availability network that
+makes it easy for anyone to securely launch their own blockchain.
+
+Celestia acts as a separate layer dedicated to storing transaction data,
+enabling app chains to focus solely on transaction execution and settlement.
+This design facilitates significant scalability
+improvements in the blockchain ecosystem.
+
+## Use Celestia in Madara
+
+When launching a Madara node, you can specify: `--da-layer celestia`.
+This will use Celestia to publish the state_diff instead of Ethereum.
+
+When launching the node, the da-config.json file path can be
+provided along the `--da-conf` flag.
+
+_**da-config.json**_
+
+```json
+{
+  "http_provider": "http://127.0.0.1:26658",
+  "ws_provider": "ws://127.0.0.1:26658",
+  "nid": "Madara",
+  "auth_token": "<see below>"
+}
+```
+
+### Launch with Celestia
+
+The following steps require the
+[Celestia CLI tool](https://docs.celestia.org/developers/node-tutorial)
+to be installed and configured
+
+### Node Initialization
+
+Run the following command to initialize a light node for the Mocha testnet:
```

Comment:
what version is this using @SamrendraS ?

Reply:
i believe the version described in the node-tutorial page is used, ver="1.21.1"

Reply:
we just need a celestia light node running to configure Madara with it's auth token

Reply:
what version of celestia-node, though? 1.21.1 is the go version

Reply:
hi, i'm sorry didn't realise that was the go version
the latest version that has been run with madara is 0.12.2

Reply:
hi @jcstein 
can you tell me if any inputs are needed from my side

Reply:
yes, it needs to be compatible with v0.12.4+, and ideally on v0.13.1

Reply:
thanks for letting me know, will be checking the compatibility with these versions and will update the PR accordingly.

Reply:
sweet, also please add the version number in doc, too so it is locked to a working version

Reply:
kind bump @SamrendraS

Reply:
hi @SamrendraS any update here?